Resources/Private/Language/PluginHaiku/de.locallang.xlf

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+<xliff version="1.0">
+	<file source-language="en" datatype="plaintext" original="messages" date="2022-09-18T18:44:59Z" product-name="examples">
+		<header/>
+		<body>
+			<trans-unit id="button.backToList" approved="yes">
+				<source>Back to list</source>
+				<target>Zurück zur Übersicht</target>
+			</trans-unit>
+			<trans-unit id="season.spring" approved="yes">
+				<source>spring</source>
+				<target>Frühling</target>
+			</trans-unit>
+			<trans-unit id="season.summer" approved="yes">
+				<source>summer</source>
+				<target>Sommer</target>
+			</trans-unit>
+			<trans-unit id="season.autumn" approved="yes">
+				<source>autumn</source>
+				<target>Herbst</target>
+			</trans-unit>
+			<trans-unit id="season.winter" approved="yes">
+				<source>winter</source>
+				<target>Winter</target>
+			</trans-unit>
+			<trans-unit id="season.theFifthSeason" approved="yes">
+				<source>Carnival</source>
+				<target>Fasching</target>
```

Comment:
```suggestion
				<target>Fastnacht</target>
```
;)

Reply:
I'll take it easy, in the northern part of BaWü we know both ;-)